known-limitations.md

```diff
@@ -248,3 +248,33 @@ Given a query like `SELECT * FROM foo WHERE a > 1 OR b > 2`, even if there are a
 ## Privileges for `DELETE` and `UPDATE`
 
 Every [`DELETE`](delete.html) or [`UPDATE`](update.html) statement constructs a `SELECT` statement, even when no `WHERE` clause is involved. As a result, the user executing `DELETE` or `UPDATE` requires both the `DELETE` and `SELECT` or `DELETE` and `UPDATE` [privileges](privileges.html) on the table.
+
+## Schema changes between executions of prepared statements
+
+When the schema of a table targeted by a `PREPARE` statement changes before the prepared statement is executed, CockroachDB should return an error. Instead, CockroachDB incorrectly allows the prepared statement to return results based on the changed table schema, for example:
```

Comment:
We need to note that this affects both kinds of prepare statements. I'm not sure this is the clearest way, but maybe:

> When the schema of a table targeted by a prepared statement or a `PREPARE` statement changes, CockroachDB should return an error the next time it is executed.